ls1intum/Artemis
# `Assessment`: Fix the assess next button when no more submissions are assessable

### Checklist
#### General
#### Client

### Motivation and Context
During the exam mode testing session we noticed that pressing assess next did not change the page when no next submission is available. 

### Description
PR #5795 changed the endpoint to return status 200 with an empty response if no submission to assess exists. But the code responsible for doing the request did not handle this case correctly, returning `{ }` instead of `undefined`.

### Steps for Testing
Prerequisites:
- 1 Tutor
- 1 Exercise of every type with submissions

1. As a tutor, assess the last student submission.
2. Press the assess next button
3. Make sure a message showing that no more submissions are available to assess appears.

#### Exam Mode Testing
same as above, just with exam exercises.

### Review Progress

#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test
